samples/snippets/sample_default_values.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A sample script showing how to handle default values when communicating
+with the Compute Engine API.
+"""
+# [START compute_instances_verify_default_value]
+# [START compute_usage_report_set]
+# [START compute_usage_report_get]
+# [START compute_usage_report_disable]
+from google.cloud import compute_v1
+# [END compute_usage_report_disable]
+# [END compute_usage_report_get]
+# [END compute_usage_report_set]
+
+
+# [START compute_usage_report_set]
+def set_usage_export_bucket(project_id: str, bucket_name: str,
+                            report_name_prefix: str = "") -> None:
+    """
+    Set Compute Engine usage export bucket for the Cloud project.
+    This sample presents how to interpret the default value for the
+    report name prefix parameter.
+
+    Params:
+        project_id: project ID or project number of the project to update.
+        bucket_name: Google Cloud Storage bucket used to store Compute Engine
+            usage reports. An existing Google Cloud Storage bucket is required.
+        report_name_prefix: Prefix of the usage report name which defaults to an empty string
+            to showcase default values behaviour.
+    """
+    usage_export_location = compute_v1.UsageExportLocation({
+        'bucket_name': bucket_name,
+        'report_name_prefix': report_name_prefix
+    })
```

Comment:
```suggestion
    usage_export_location = compute_v1.UsageExportLocation(
        bucket_name=bucket_name,
        report_name_prefix=report_name_prefix
    )
```

Constructing it this way allows for IDE code completion to help out.

Reply:
OK